config.py

```diff
@@ -1551,7 +1551,7 @@ def get_runs(subject: str, verbose: bool = False) -> Union[List[str], List[None]
 
 
 # XXX This check should actually go into the CHECKS section, but it depends
-# XXX on get_runs(), which is defined after that section.
+# XXX on get_runs(subject=subject), which is defined after that section.
```

Comment:
Can you please undo this change? In the comment here we left out the parameters on purpose – after all, we just wanted to point out a function name